### ISSUE TITLE
ionuttbara copy your bat script removing Edge, rename it as own project and share via Majorgeeks

### ISSUE THREAD

**Issue by FadeMind** [AveYo/fox#33]
Hi @AveYo 

Majorgeeks admin share sometimes projects from Github. 
I recently found this https://www.majorgeeks.com/files/details/edge_remover.html

![20230806-1691351342](https://github.com/AveYo/fox/assets/9846948/afb59a46-490d-4b08-8604-319cb5c34db6)

Works well. Anyway, https://github.com/jbara2002 is not valid anymore. 

After digging I found > https://github.com/ionuttbara/edge-remover/tree/main

but self release does not contain source code. https://github.com/ionuttbara/edge-remover/releases/tag/release2.7
EXE is just 7Z SFX.
![20230806-1691351088](https://github.com/AveYo/fox/assets/9846948/fe8df9a5-7757-4efc-9f62-114bae19dad1)

And yes, this script removing Edge, not Onedrive this time... 

@ionuttbara just copy paste your work here and create SFX with 7Z. 

Difference is one. Disabled removing Edge Web View compotents.

**Comment by AveYo** [comment]
Even ChrisTitus was bundling a stripped tag version. What can you do...
Now I see that it's updated with the current one so that's great!
In the end, the more people resisting the edge deathtrap, the more happy it makes me.
And I'm glad it gets included in projects with a larger scope and reach.
Maybe my lean and mean style will rub off on others - I hate scroll-for-days-until-you-forget-where-it-started bloated comment-every-uninterested-line microsoft style.

As for those sfx out of perfectly fine plain text scripts - I find them backwards and in bad taste.

Thank you for your support! Just don't waste any of your time being bothered by it.

**Comment by rcmaehl** [comment]
Yeah ChrisTitus' implementation is fine. As long as someone is using an updated script, no matter who made it, it's good!

**Comment by FadeMind** [comment]
@AveYo 
![20230915-1694789984](https://github.com/AveYo/fox/assets/9846948/606ccba6-3b38-47a2-89fb-53b6bf93e590)

**Comment by AveYo** [comment]
> 
> Hi. I use that implementation, because the old version doesn't remove some Edge's Registry Entires, so i found this. I use this for to make confortable, and not to copy or another things. I don't know the projects (not all) are uploaded to MajorGeeks. So, this is good news for me. (Defender Remover and OneDrive Remover are not in that website. 😭) From the quote, i modified that script. I updated today in my edge removal repository, with some modifications (renamed into .ps1 file, added chrome downloads, and DoNotForceUpdateWithChormium policy which you can found there ([ Winaero Article Link ](https://winaero.com/prevent-microsoft-edge-chromium-from-installing-via-windows-update/)). If @AveYo have problems in script usage, and to develop in my repository, he can contact me. 😊

No, it's not me having to contact you :)
Weird quote above, since DoNotForceUpdateWithChormium has been used in the script since forever. Microsoft just ignores it _(so you might want to rebase from the latest, as I've implemented a soft-block for Edge and improved cleanup)_.
If you managed to understand the script and implement your own small changes without breaking anything, that's something that takes effort and nobody is gonna hold it against you.
But you have to admit that taking a plain-text script and turn it into an exe calling it your own was a lame move. That's all.
